typings/rendition.d.ts

```diff
@@ -508,4 +496,65 @@ declare module 'rendition' {
 	}
 
 	const Theme: Theme;
+
+	namespace v3 {
```

Comment:
Could we move this to an other file and re-export as part of the namespace here?
Or maybe we can also declare `declare module 'rendition' { namespace V3 { /*...*/ } }` in the other file and just import it here.

Reply:
I prefer to keep it here for simplicities sake